pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -145,6 +145,18 @@ type RevisionSpec struct {
 	// https://github.com/knative/serving/issues/627
 	// +optional
 	Container corev1.Container `json:"container,omitempty"`
+
+	// NodeSelector is a selector which must be true for the revision's pod(s) to
+	// fit on a node.
+	// Selector which must match a node's labels for the pod to be scheduled on
+	// that node.
+	// More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
This api change (and the Tolerations below it) should include a corresponding change to the spec at https://github.com/knative/serving/blob/master/docs/spec/spec.md and the conformance tests at https://github.com/knative/serving/tree/master/test/conformance, if applicable.

Reply:
@bbrowning I updated docs just now.

I'm really not sure whether the conformance tests are applicable here, and if so, I'm not sure where to begin.